2021/01/13 14:36:35 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Arquivo ou diret처rio n찾o encontrado in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /opt/lampp/htdocs/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /opt/lampp/htdocs/yii/framework/gii/CCodeGenerator.php(154): CFormModel->__construct()
#7 /opt/lampp/htdocs/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /opt/lampp/htdocs/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /opt/lampp/htdocs/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /opt/lampp/htdocs/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(283): CController->run('')
#13 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(142): CWebApplication->runController('gii/model')
#14 /opt/lampp/htdocs/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /opt/lampp/htdocs/yii/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/blog/index.php?r=gii/default/index
---
2021/01/13 14:38:17 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Arquivo ou diret처rio n찾o encontrado in /opt/lampp/htdocs/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /opt/lampp/htdocs/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /opt/lampp/htdocs/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /opt/lampp/htdocs/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /opt/lampp/htdocs/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /opt/lampp/htdocs/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /opt/lampp/htdocs/yii/framework/gii/CCodeGenerator.php(154): CFormModel->__construct()
#7 /opt/lampp/htdocs/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /opt/lampp/htdocs/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /opt/lampp/htdocs/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /opt/lampp/htdocs/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /opt/lampp/htdocs/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(283): CController->run('')
#13 /opt/lampp/htdocs/yii/framework/web/CWebApplication.php(142): CWebApplication->runController('gii/model')
#14 /opt/lampp/htdocs/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /opt/lampp/htdocs/yii/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/blog/index.php?r=gii/default/index
---
2021/01/13 14:42:17 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No such file or directory in /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /home/conexa-dev-02/Documentos/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /home/conexa-dev-02/Documentos/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /home/conexa-dev-02/Documentos/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/conexa-dev-02/Documentos/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /home/conexa-dev-02/Documentos/yii/framework/gii/CCodeGenerator.php(154): CFormModel->__construct()
#7 /home/conexa-dev-02/Documentos/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /home/conexa-dev-02/Documentos/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(283): CController->run('')
#13 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(142): CWebApplication->runController('gii/model')
#14 /home/conexa-dev-02/Documentos/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /home/conexa-dev-02/Documentos/yii/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://127.0.0.1:8080/index.php?r=gii/default/index
---
2021/01/13 14:43:09 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No such file or directory in /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /home/conexa-dev-02/Documentos/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /home/conexa-dev-02/Documentos/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /home/conexa-dev-02/Documentos/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/conexa-dev-02/Documentos/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /home/conexa-dev-02/Documentos/yii/framework/gii/CCodeGenerator.php(154): CFormModel->__construct()
#7 /home/conexa-dev-02/Documentos/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /home/conexa-dev-02/Documentos/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(283): CController->run('')
#13 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(142): CWebApplication->runController('gii/model')
#14 /home/conexa-dev-02/Documentos/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /home/conexa-dev-02/Documentos/yii/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://127.0.0.1:8080/index.php?r=gii/default/index
---
2021/01/13 15:13:57 [error] [exception.CDbException] CDbException: CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No such file or directory in /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /home/conexa-dev-02/Documentos/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /home/conexa-dev-02/Documentos/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /home/conexa-dev-02/Documentos/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /home/conexa-dev-02/Documentos/yii/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/conexa-dev-02/Documentos/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /home/conexa-dev-02/Documentos/yii/framework/gii/CCodeGenerator.php(154): CFormModel->__construct()
#7 /home/conexa-dev-02/Documentos/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /home/conexa-dev-02/Documentos/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(283): CController->run('')
#13 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(142): CWebApplication->runController('gii/model')
#14 /home/conexa-dev-02/Documentos/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /home/conexa-dev-02/Documentos/yii/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://127.0.0.1:8080/index.php?r=gii
---
2021/01/13 15:46:58 [error] [exception.ParseError] ParseError: syntax error, unexpected '$private' (T_VARIABLE), expecting function (T_FUNCTION) in /home/conexa-dev-02/Documentos/yii/blog/protected/components/UserIdentity.php:19
Stack trace:
#0 [internal function]: YiiBase::autoload('UserIdentity')
#1 /home/conexa-dev-02/Documentos/yii/blog/protected/models/LoginForm.php(51): spl_autoload_call('UserIdentity')
#2 /home/conexa-dev-02/Documentos/yii/framework/validators/CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#3 /home/conexa-dev-02/Documentos/yii/framework/validators/CValidator.php(214): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#4 /home/conexa-dev-02/Documentos/yii/framework/base/CModel.php(158): CValidator->validate(Object(LoginForm), Array)
#5 /home/conexa-dev-02/Documentos/yii/blog/protected/controllers/SiteController.php(94): CModel->validate()
#6 /home/conexa-dev-02/Documentos/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#7 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#8 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#9 /home/conexa-dev-02/Documentos/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(283): CController->run('login')
#11 /home/conexa-dev-02/Documentos/yii/framework/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#12 /home/conexa-dev-02/Documentos/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#13 /home/conexa-dev-02/Documentos/yii/blog/index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://127.0.0.1:8080/index.php?r=site/login
---
